drivers/bridge/setup_ip_tables.go

```diff
@@ -149,7 +149,7 @@ func setIcc(bridgeIface string, iccEnable, insert bool) error {
 			iptables.Raw(append([]string{"-D", chain}, dropArgs...)...)
 
 			if !iptables.Exists(table, chain, acceptArgs...) {
-				if output, err := iptables.Raw(append([]string{"-A", chain}, acceptArgs...)...); err != nil {
```

Comment:
@alecbenson Wouldn't the `-A => -I` change be needed at line 142 as well ? if not, can you please explain why. Thanks.